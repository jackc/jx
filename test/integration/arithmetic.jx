var firstname string
firstname = "John"
var lastname string
lastname = "Smith"
var name string
name = firstname + " " + lastname
puts(name + "\n")

var n int
n = 7
var m int
m = 3

puti(n+m)
puts("\n")

puti(n-m)
puts("\n")

puti(n*m)
puts("\n")

puti(n/m)
puts("\n")
