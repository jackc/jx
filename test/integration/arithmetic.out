John Smith
10
4
21
2
